packages/core/src/App/Containers/Modals/cooldown-warning-modal.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import { Button, Modal, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { formatDate } from '@deriv/shared';
+
+const CooldownWarningModal = ({
+    should_show_cooldown_modal,
+    setShouldShowCooldownModal,
+    real_account_creation_unlock_date,
+}) => {
+    const real_account_unblock_date = formatDate(real_account_creation_unlock_date, 'DD MMMM [at] hh:mm a');
```

Comment:
How will this `at` get translated?

Reply:
The text provided as part of string format gets translated as per the moment configured language since The moment instance is configured to the selected language